dnf/cli/cli.py

```diff
@@ -861,6 +861,7 @@ def _parse_commands(self, opts, args):
         basecmd = opts.command[0]
         command_cls = self.cli_commands.get(basecmd)
         if command_cls is None:
+            self.optparser.print_help()
```

Comment:
I think there is no need to print help if user just make a typo and the consequent "Please use --help" also does not make sense then.